python/src/remotebmi/client/client.py

```diff
@@ -48,17 +58,18 @@ def finalize(self):
     def get_component_name(self):
         response = self.client.get("/get_component_name")
         response.raise_for_status()
+        # TODO validate response, with pydantic or similar, should be done for all responses
```

Comment:
Can you convert this TODO to an issue? Would be easier to keep track of it that way

Reply:
Opened issue